### PR TITLE
Add FTB App manifest

### PR DESCRIPTION
**Pull request by palmtree5**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/847)

### CODE REVIEW

File: manifests/FTB/FTB/1.0.8.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Id: FTB.FTB
+Version: 1.0.8
+Name: FTBApp
+Publisher: Feed The Beast
+License: LGPL 2.1
```
Comment (jamierocks) [inline manifests/FTB/FTB/1.0.8.yaml:5]: While this is true for the frontend, its not true for the backend (https://github.com/CreeperHost/modpacklauncher) which the installer is built from.

**Review by jamierocks — CHANGES_REQUESTED** [review]

File: manifests/FTB/FTB/1.0.8.yaml (hunk 1 above)
Comment (palmtree5) [reply manifests/FTB/FTB/1.0.8.yaml]: Fair point, I updated it to mention the licenses for both the frontend and the backend

**Review by jamierocks — CHANGES_REQUESTED** [review]
Are there any switches to do a silent install?

**Comment by palmtree5** [comment]
not that I could find

File: manifests/FTB/FTB/1.0.8.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+License: LGPL 2.1 (frontend), CC BY-NC-ND (backend)
+LicenseUrl: https://github.com/FTBTeam/FTB-App/blob/master/LICENSE.md
+AppMoniker: ftb
+Tags: game, minecraft, ftb
+Homepage: https://www.feed-the-beast.com/
+Installers:
+  - Arch: x64
+    Url: https://apps.modpacks.ch/FTBApp/release/1.0.8/FTBA_windows-x64_1_0_8.exe
+    Sha256: 79372BA75C585D1F4DBE288C71591735FD288E2C821CF94940A8CC2551C04948
+    InstallerType: exe
```
Comment (jamierocks) [inline manifests/FTB/FTB/1.0.8.yaml:14]: Do we know what installer they use? nullsoft, etc

**Review by jamierocks — CHANGES_REQUESTED** [review]

File: manifests/FTB/FTB/1.0.8.yaml (hunk 2 above)
Comment (palmtree5) [reply manifests/FTB/FTB/1.0.8.yaml:14]: After reading install4j's docs, I figured out that it's `-q`.

**Review by ghost — APPROVED** [review]
Validation has completed

**Pull request merged**